packages/ml-kit/README.md

```diff
@@ -0,0 +1,156 @@
+> In other words, the main application is the one in charge of executing the required data preprocessing techniques (e.g., normalization, feature extraction, etc...) to prepare the data for the model, not this module.
+
+To register these tasks for their use, you just need to import them and call their generator functions inside your application's task list:
+
+```typescript
+import { Task } from '@awarns/core/tasks';
+import {
+  classificationTask,
+  regressionTask,
+  DelegateType,
+} from '@awarns/ml-kit';
+import { DelegateType } from './index';
+
+export const demoTasks: Array<Task> = [
+  classificationTask('human-activity', 'activity_classifier-mlp', '', { acceleration: 4 }),
+  // humanActivityClassification
+
+  classificationTask('human-activity', 'activity_classifier-cnn', 'UsingCNN', { acceleration: DelegateType.GPU }),
+  // humanActivityClassificationUsingCNN
+  
+  regressionTask('stress-level', 'stress_regressor-cnn'),
+  // stressLevelRegression
+]
+```
+
+**Task generator parameters:**
+
+| Parameter name                                   | Type                                                  | Description                                                                                                                                                         |
+|--------------------------------------------------|-------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <code>{classification&vert;regression}Aim</code> | `string`                                              | Objective of the classification/regression. Used to name the task.                                                                                                  |
+| `modelName`                                      | <code>string &vert; ModelNameResolver</code>          | Name of the model (without *tflite* extension) stored in the *ml-models* folder to use for this task, or a function that returns the name of the model when called. |
+| `tag` *(Optional)*                               | `string`                                              | Adds a tag to the name of the task to differentiate it from other tasks with other configurations.                                                                  |
+| `modelOptions` *(Optional)*                      | <code>ModelOptions &vert; ModelOptionsResolver</code> | Configuration to use with the model or a function that returns the configuration when called.                                                                       |
+
+- `ModelNameResolver: () => string`
+
+Useful to change the model used by a task at runtime. You can use the [`ModelManager`](#modelmanager) to obtain a list with the models that are available in the device.
```

Comment:
Oh, I see it's here too. I don't know what's the best place to put it, to be honest

Reply:
Yeah, me neither. It could be in both places 🤔